### PR TITLE
EN: Translate Episode 3 "Tricking and Deceiving Katori"

### PR DESCRIPTION
**Pull request by SonuOseaner**
Episode 3, Chapter 5, Sub Event (Katori & Saga), "Tricking and Deceiving Katori"

### CODE REVIEW

**Comment by alama** [comment]
You need to move this file out of the Files folder

**Comment by alama** [comment]
Look like the spelling check script is broken, let fix that...

File: Story/Episode 3/un_035010.csv [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+un_035010_0240#0,"""You don't really mean those words."""
+un_035010_0250#0,"""I certainly do!"""
+un_035010_0260#0,"""I see..."""
+un_035010_0270#0,"""Then I won't hold back. Our training thus far has only been a warm up. The real thing starts now."""
+un_035010_0280#0,"""With our next training, I'll test your new resolve!"""
+un_035010_0290#0,"""I have already prepared it. Come on, let's go."""
+un_035010_0300#0,"""...Excuse me? Already prepared, you say?"""
+un_035010_0310#0,"""B-but, hold on, Sir Saga. Shouldn't we start taking it easy...?"""
+un_035010_0320#0,"""I knew you didn't mean those words."""
+un_035010_0330#0,"""It is true my morale was high throughout the conversation, b-but...!"""
```
Comment (DabirSA) [inline Story/Episode 3/un_035010.csv]: Don't like this last line. It's a bit awkward, but I'm not sure how to rewrite it.

File: Story/Episode 3/un_035010.csv [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+un_035010_0070#0,"""So harsh! Unemotional as usual!"""
+un_035010_0080#0,"""Aah~ If only I could be partnered up with you instead of Saga."""
+un_035010_0090#0,"""...I understand. If that's how you feel, it can't be helped. Would you rather try it that way?"""
+un_035010_0100#0,"""...Eh?"""
+un_035010_0110#0,"""I've reached the limits of what I can teach, I'm afraid."""
+un_035010_0120#0,"""I'll take care of the paperwork. Your training will remain the same, but you will have a new partner."""
+un_035010_0130#0,"""Ah? N-no...<br>Sir Saga? What are you..."""
+un_035010_0140#0,"""<%me>, my apologies.<br>From now on, you'll have to take care of her. I'm counting on you."""
+un_035010_0150#0,"""Ho-hold on! Wait! Sir Saga, I was jesting! A joke! I'm sure you understand!"""
+un_035010_0160#0,"""...Don't overstrain yourself, Katori. After all our training, your body's in a lot of pain, isn't it?"""
```
Comment (alama) [inline Story/Episode 3/un_035010.csv:18]: is "overstrain" a misspelling?

File: _aspell/PSO2.en_Chat.dict [hunk 3]
```diff
@@ -866,3 +866,4 @@ callin
 Hwahahaha
 Hwahahahahaha
 Hmhmhm
+overstrain
```
Comment (alama) [inline _aspell/PSO2.en_Chat.dict:870]: Should make a rule that if we going to update `_aspell` files, do a upstream merge first?

**Pull request merged**